.rubocop.yml

```diff
@@ -8,6 +8,15 @@ AllCops:
     - 'config/**/*'
     - 'vendor/**/*'
 
+Style/MethodCallWithArgsParentheses:
+  Enabled: true
+  Exclude:
+    - 'config.ru'
+  IgnoredMethods:
+    - require
+    - to
+    - to_not
```

Comment:
@brindu au fait pourquoi favoriser `to_not` plutôt que `not_to` sachant qu'en Anglais "not to" est beaucoup plus courant ?

Reply:
Je préfère juste le côté *catchy* de *to not* dans le cas particulier des specs. 

*Expect to not ... !* je trouve qu'il y a un côté plus "sec", indiscutable (j'ai du mal à trouver les mots pour décrire), à la manière dont on donnerait un ordre, qui, je trouve colle bien dans le contexte de l'écriture des specs.

Là où *Expect not to...*, je trouve que l'intonation induit un petit peu de laxisme, beaucoup moins impérative.

En fait, dans le premier cas je mettrais un point d'exclamation : *Expect to not!* alors que dans le deuxième cas *Expect not to!* je trouve que ça ne marche pas ; et j'aime bien le côté "impératif" dans les specs. 

Fondamentalement les deux marchent et je voulais juste pas rajouter *not_to* dans la liste des méthodes ignorées.

Reply:
oui je vois ce que tu veux dire, en effet c'est plus impératif. Je m'y habituerai

Reply:
pour une fois je suis direct d'accord avec les goûts de @brindu